SmartDeviceLink/SDLHapticManager.m

```diff
@@ -0,0 +1,161 @@
+//
+//  SDLHapticManager.m
+//  SmartDeviceLink-iOS
+//
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "SDLHapticManager.h"
+#import "SDLNotificationConstants.h"
+#import "SDLRectangle.h"
+#import "SDLHapticRect.h"
+#import "SDLSendHapticData.h"
+#import "SDLTouch.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLHapticManager()
+
+/**
+ The projection window associated with the Haptic Manager
+ */
+@property (nonatomic, strong) UIWindow *projectionWindow;
```

Comment:
Should this be weak because we aren't the primary owner of the window (i.e. is this window expected to be owned by us or by the developer)?